cli/ballerina-packerina/spotbugs-exclude.xml

```diff
@@ -25,6 +25,14 @@
         <Class name="org.ballerinalang.packerina.model.BaloToml"/>
         <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
     </Match>
+    <Match>
+        <Class name="org.ballerinalang.packerina.cmd.InitCommand"/>
+        <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
```

Comment:
Lets get rid of this exclusion using `java.util.Optional`

Reply:
This is added here because of the line L115 in InitCommand. It expects that statement may lead to a NPE. However, it would not be null.